kolibri/core/device/models.py

```diff
@@ -80,15 +80,16 @@ def app_is_enabled():
     return interface.enabled
 
 
+# '"optional":True' is obsolete but needed while we keep using an
+# old json_schema_validator version compatible with python 2.7
```

Comment:
Maybe also expand and mention that this needs to be backwards compatible? I'm thinking it would be a good thing given I'm guessing your migration here will stop Django wanting to create more migrations for schema changes?